ophyd/v2/core.py

```diff
@@ -788,57 +787,6 @@ async def _wait_for_value():
     await asyncio.wait_for(_wait_for_value(), timeout)
 
 
-@runtime_checkable
-class AsyncReadable(Protocol):
-    """Readable narrowed to only be async"""
-
-    @abstractmethod
-    async def read(self) -> Dict[str, Reading]:
-        ...
-
-    @abstractmethod
-    async def describe(self) -> Dict[str, Descriptor]:
-        ...
```

Comment:
so is the idea that now, all readables are async in v2?

Reply:
I simplified `StandardReadable` so it only reads `SignalR`s, which meant I no longer needed this intermediate protocol for "things that `StandardReadable` can read"